blog/2024-12-04-area-squared-units.md

```diff
@@ -0,0 +1,25 @@
+---
+author: mikey0000
+authorURL: https://github.com/mikey0000
+title: "Area squared unit enumerators"
+---
+
+As of Home Assistant Core 2024.12, the following unit constant is deprecated and replaced by a corresponding enum:
```

Comment:
I think the focus should be on the new features (the deprecation is a side-effect)
```suggestion
## Summary of changes
A new `AREA` device class is now available for number and sensor entities, together with automatic unit conversion based on the unit system.
A corresponding `UnitOfArea` unit enumerator, and `AreaConverter` converter class have been added to support the new device class.

As of Home Assistant Core 2024.12, the following unit constant is deprecated and replaced by a corresponding enum:
```